app/styles/_projects.less

```diff
@@ -71,8 +71,8 @@
     }
     a {
       display: block;
+      line-height: normal;
```

Comment:
@MarkDeMaria, any chance you remember why?  https://github.com/openshift/origin-web-console/blame/master/app/styles/_projects.less#L75